luigi/contrib/kubernetes.py

```diff
@@ -213,10 +213,15 @@ def poll_interval(self):
         """How often to poll Kubernetes for job status, in seconds."""
         return self.__DEFAULT_POLL_INTERVAL
 
+    @property
+    def pod_creation_wait_interal(self):
+        """Delay for initial pod creation for just submitted job"""
```

Comment:
Please use a constant similar as `__DEFAULT_POLL_INTERVAL`, also in docstring please describe the unit so it would be easier to follow.